cmd/soci/commands/create.go

```diff
@@ -54,6 +55,10 @@ var CreateCommand = cli.Command{
 			Usage: "Minimum layer size to build zTOC for. Smaller layers won't have zTOC and not lazy pulled. Default is 10 MiB.",
 			Value: 10 << 20,
 		},
+		cli.BoolFlag{
+			Name:  xattrFlag,
```

Comment:
Other flag names have the whole cli flag name duplicated. I think we should be consistent.
```suggestion
			Name:  xattrOptimizationFlag,
```